src/TulsiGenerator/XcodeProjectGenerator.swift

```diff
@@ -943,6 +943,7 @@ final class XcodeProjectGenerator {
                                  project: info.project,
                                  projectBundleName: projectBundleName,
                                  testActionBuildConfig: runTestTargetBuildConfigPrefix + "Debug",
+                                 launchActionBuildConfig: runTestTargetBuildConfigPrefix + "Debug",
```

Comment:
Excuse me because this sneeze attack I have is giving me brain fog – but does this fix using Cmd-B for test targets as well?

Reply:
Yup, it does.